<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert3</class>
 <widget class="QWidget" name="insert3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>770</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入文件地址：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>600</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>210</y>
     <width>112</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>请点击</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
